package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "heroku-pg-extras",
   "description": "A Heroku CLI plugin for awesome pg:* commands that are also great and fun and super.",
-  "version": "1.0.11",
+  "version": "1.0.12",
```

Comment:
Not that we follow semver all that carefully, but adding a new command should probably be a minor version bump to 1.1.0...

Reply:
changed to 1.1.0